src/components/MentionTextArea/MentionList.tsx

```diff
@@ -21,91 +26,98 @@ export const SuggestionLoading = () => {
   );
 };
 
-// Handler dropdown scroll event on keypress
+/** Handler dropdown scroll event on keypress */
```

Comment:
curiosity - is this a stylistic preference? or something else?

Reply:
Comments wrapped in `/** */` show up when you hover over a reference to the variable (at least in VS Code), but `//` comments don't!

I generally use `/** comment */` for documentation that explains what something does or gives info that's useful for implementation. If it's a comment that's only relevant to the initial code, I use `// comment`.

So like in [this code block](https://github.com/CMS-Enterprise/easi-app/blob/EASI-4658/integrate-tagging-discussions/src/components/MentionTextArea/index.tsx#L28-L41) I use both because only the top comment is helpful if I'm reading code later that uses that function and want to know what it's doing.

```jsx
/** The rendered Mention after selected from MentionList */
// This component can be any react jsx component, but must be wrapped in <NodeViewWrapper />
const MentionComponent = ({ node }: NodeViewProps) => {
```

Reply:
oh wow! that is really interesting - thank you for the info!

Reply:
No problem!